core/src/main/java/io/grpc/ConnectivityStateInfo.java

```diff
@@ -94,6 +94,10 @@ public int hashCode() {
     return state.hashCode() ^ status.hashCode();
   }
 
+  /**
+   * If the {@link Status} is OK, then return the {@link ConnectivityState}. If there is an error,
+   * then return both.
+   */
```

Comment:
In general `toString()` is left undocumented, unless you want to strictly define its format so that it can be consumed programmaticly.